sdks/java/core/src/main/java/org/apache/beam/sdk/util/ApiSurface.java

```diff
@@ -321,25 +321,26 @@ public void describeTo(final Description description) {
   ///////////////
 
   /** Returns an empty {@link ApiSurface}. */
-  public static ApiSurface empty() {
+  private  ApiSurface empty() {
```

Comment:
This is a side effect of making the class abstract, since it was constructing an object.